packages/wrangler/src/d1/delete.ts

```diff
@@ -30,6 +32,7 @@ export const d1DeleteCommand = createCommand({
 	},
 	positionalArgs: ["name"],
 	async handler({ name, skipConfirmation }, { config }) {
+		printResourceLocation("remote");
```

Comment:
It prints `Resource location: remote` and then takes a while to do the requireAuth() and getDatabaseByNameOrBinding() calls before printing anything else or prompting for confirmation, which is a bit disconcerting. 

Can we move this down so it prints immediately before the "About to delete" log line, below?